pkg/controller/plan/migration.go

```diff
@@ -649,6 +651,23 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 		}
 
 		var pvcNames []string
+		var dataVolumes []cdi.DataVolume
```

Comment:
why moving it here? I'd also keep pvcNames closer to where it's used below

Reply:
I think because in some revision I used it in the block below, but looks like I don't do that anymore 😅